language/diem-framework/modules/DiemAccount.move

```diff
@@ -506,7 +506,7 @@ module DiemAccount {
         assert(MinerState::can_create_val_account(sender_addr), Errors::limit_exceeded(120102));
         // Check there's enough balance for bootstrapping both operator and validator account
         assert(
-            balance<GAS>(sender_addr) >= 2 * BOOTSTRAP_COIN_VALUE, 
+            balance<GAS>(sender_addr) > 2 * BOOTSTRAP_COIN_VALUE, 
```

Comment:
@0o-de-lally 
Does your previous comment about line 1440 "We generally don't want people to have zero balances after an operation. It makes it hard to recover from a problem if a tx needs to be sent and there is no gas." apply here too ? If so, that's my change about.

Reply:
Yes it should apply here as well. Good catch.